catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/content/operation/impl/CreateStorageResponseImpl.java

```diff
@@ -60,4 +62,16 @@ public CreateStorageResponseImpl(CreateStorageRequest request,
     public StorageRequest getStorageRequest() {
         return request;
     }
+
+    @Override
+    public String toString() {
+        if (createdContentItems == null) {
+            return "CreateStorageResponseImpl{createdContentItems=null}";
+        }
+        return String.format("CreateStorageResponseImpl{createdContentItems=%s}",
+                createdContentItems.stream()
+                        .filter(Objects::nonNull)
+                        .map(Object::toString)
```

Comment:
The more wholesome solution is to do exactly that. A utility class that interacts with the interface of an object to reliably produce a textual representation... However that is far exceeding the scope of this ticket. Since we are in control of our own implementations this should be fine for the common use case. Those that are using their own implementations also supply a toString or they will simply lose some of the benefit this message provides.